datastore/api/data.py

```diff
@@ -42,9 +42,24 @@ class InMemoryDataset(Dataset):
     """ Abstract class for in memory data """
 
     def load_data(self):
+        """ Load data and labels """
         raise NotImplementedError
 
 
+class MultiTaskMeta(type):
+    """ Metaclass for Multitask Datasets """
+
+    def __init__(cls, name, bases, dct):
+        cls.labels = {}
```

Comment:
Imposes some structure on base Multitask datasets. Shows that labels should be held in dictionaries.